Dockerfile

```diff
@@ -53,3 +52,35 @@ COPY extend-environment.sh /etc/profile.d/extend-environment.sh
 RUN pip install --no-cache-dir -r requirements/local.txt
 
 CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
+
+FROM cfgov-develop as cfgov-build
+
+# add node and yarn repos
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+
+RUN yum install -y nodejs yarn  && \
+    yum clean all && rm -rf /var/cache/yum
+
+WORKDIR /src/cfgov-refresh
```

Comment:
This is already set above.

https://github.com/cfpb/cfgov-refresh/pull/5177/files#diff-3254677a7917c6c01f55212f86c57fbfR8-R10

Reply:
removed!